api/src/main/java/org/openmrs/module/referenceapplication/ReferenceApplicationActivator.java

```diff
@@ -105,6 +107,8 @@ public void started() {
             setupTagLocation(ReferenceMetadataConstants.VISIT_LOCATION_TAG_UUID);
 	        setupHtmlForms();
 	        setupHL7ProcessingTask(schedulerService);
+
+			ReportManagerUtil.setupAllReports(ReportManager.class);
```

Comment:
Could you please add a comment saying that it sets up reports from org.openmrs.module.referenceapplication.reports?